apps/meteor/app/api/server/ApiClass.ts

```diff
@@ -171,7 +171,7 @@ export class APIClass<
 		inviteToken: number;
 	};
 
-	readonly router: Router<any>;
+	readonly router: RocketChatAPIRouter<any>;
```

Comment:
I think we can keep `Router` interface usega around... no need to replace all the places. unless in some specific case you really need some method that is only implemented  at `RocketChatAPIRouter`